### DISCUSSION TITLE
How do I bind the value of a dropdown selection (Options-Picker)?

### DISCUSSION THREAD

**Discussion by enpro-github** [Budibase/budibase#4168] [Help]
I've been trying to figure this out for days but every time I try to reference my field in my dropdown menu nothing appears in the query that uses said binding. 

![image](https://user-images.githubusercontent.com/68210951/150812214-b716d0f8-6b01-4c69-acfd-839613db49eb.png)

The field name for "Utility Picker" is called "utility_name"

Do I reference it with `Form.Fields.utility_name` ? Or do I need to reference another level, like with `Form.Fields.Dropdown.utility_name`? Because it's having no effect

Thank you

**Comment by mjashanks** [comment]
It should be `Form.Fields.utility_name`

It should appear in your list of bindings - I just tried it out:

![image](https://user-images.githubusercontent.com/3524181/150814920-8b5c29c0-462a-4cff-b5a5-f12adce4ddbc.png)

I was able to bind the value to a `Paragraph` component:

![image](https://user-images.githubusercontent.com/3524181/150815267-59f8a793-bfc9-4901-859d-f8ef0afda712.png)

**Reply by enpro-github** [reply]
> It should be `Form.Fields.utility_name`
> 
> It should appear in your list of bindings - I just tried it out:

Hey thank you for the reply! Yes I've been trying that but to no avail, when I check the inspect menu in Google Chrome the string is empty. If I bind that field to another one as a test it works however:

![image](https://user-images.githubusercontent.com/68210951/150817897-78f62aec-dfd6-4325-b7fe-953234dc3587.png)

![image](https://user-images.githubusercontent.com/68210951/150817991-22b335aa-c397-4bed-b540-e1171fda1921.png)

**Reply by mjashanks** [reply]
Thanks - I've tried to reproduce this, but cannot. I'm very interested though, because you aren't the first person today to report an issue with `Execute Query` + Parameters.

What type of data source are you using?

I've created a very simple app
1. Has an options picker with 2 GitHub orgs - Budibase & Gastby
2. A button that fetches repos for that org, from the GitHub API
3. A paragraph that displays the result.

Here is my export - can you check if it works for you? 

[Repos-export-1643041198761.txt](https://github.com/Budibase/budibase/files/7927068/Repos-export-1643041198761.txt)

**Reply by mjashanks** [reply]
Demo of my app:

https://user-images.githubusercontent.com/3524181/150822753-bdcb74a0-6f93-4644-a270-2c1f1c3b4a8d.mp4

**Reply by enpro-github** [reply]
Currently using a REST datasource

![image](https://user-images.githubusercontent.com/68210951/150823041-5f4028f0-d038-454d-bba9-0adf6cb4025c.png)


I'll give your demo a try and report back

**Reply by enpro-github** [reply]
Hmm It looks like I'm getting an error with your demo:

![image](https://user-images.githubusercontent.com/68210951/150825137-d3bb8e42-63bf-40f4-88fe-9c27d61dcf9d.png)

I'm also running version 0.9.173 and it's been having trouble updating (I have another thread with you that you havent replied to, assumed you were busy)

**Comment by enpro-github** [comment]
I figured out the issue, it looks like I added an extra layer there. The "dropdown" layer. Weird though because Budibase was still suggesting the same attribute references as if it wasn't there

Thank you again
